multi_host_migration/tests/cfg/multi_host.cfg

```diff
@@ -518,6 +518,11 @@
                     virtio_scsi:
                         no Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2
                         bus_extra_params_image1 = "iothread=${iothreads}"
+                        Windows:
```

Comment:
- with_data_plane is only for Linux in multi_host.cfg, so no need to set limitation for windows here,

Reply:
yeah, you are right, updated, thanks !